core/object/message_queue.cpp

```diff
@@ -481,19 +481,19 @@ void CallQueue::statistics() {
 		}
 	}
 
-	print_line("TOTAL PAGES: " + itos(pages_used) + " (" + itos(pages_used * PAGE_SIZE_BYTES) + " bytes).");
-	print_line("NULL count: " + itos(null_count));
+	fprintf(stdout, "TOTAL PAGES: %s (%s bytes).\n", itos(pages_used).utf8().get_data(), itos(pages_used * PAGE_SIZE_BYTES).utf8().get_data());
+	fprintf(stdout, "NULL count: %s.\n", itos(null_count).utf8().get_data());
```

Comment:
Maybe we don't need to jump through so many hoops and back? ;)
```suggestion
	fprintf(stdout, "TOTAL PAGES: %d (%d bytes).\n", pages_used, pages_used * PAGE_SIZE_BYTES);
	fprintf(stdout, "NULL count: %d.\n", null_count);
```
etc.

Reply:
I can improve it everywhere, just copied the error cases :) will change